packages/wonder-blocks-modal/src/components/__docs__/modal-dialog.stories.js

```diff
@@ -46,10 +46,26 @@ const customViewports = {
 export default {
     title: "Modal/Building Blocks/ModalDialog",
     component: ModalDialog,
+    decorators: [
```

Comment:
**issue:** It looks like a change made here broke the styles of the `ModalDialog` stories. I only see a blank canvas.

https://5e1bf4b385e3fb0020b7073c-rdqgrtjlwm.chromatic.com/?path=/docs/modal-building-blocks-modaldialog--default

<img width="1055" alt="Screen Shot 2022-09-08 at 9 41 45 AM" src="https://user-images.githubusercontent.com/843075/189137618-b2d76cda-311f-43a3-ada2-b96c576c3ac3.png">

Reply:
Wow, I have no idea how that happened. I just pushed a fix for it though.